### DISCUSSION TITLE
Suggestion: included but toggleable "DontEatMyContent" tweak

### DISCUSSION THREAD

**Discussion by machineonamission** [qnblackcat/uYouPlus#484] [Ideas]
https://reddit.com/r/jailbreak/comments/xkc72m/free_release_donteatmycontent_prevent_the/

**Comment by arichornlover** [comment]
I did try to implement this to my version of uYouPlus but that ended up breaking it due to code problems, I wasn’t skilled enough for that.

**Reply by arichornlover** [reply]
I’ll pull request this to qnblackcat, it took me 15+ Hours to make this work but glad I did it for HexCodeFFF
